--- truevision-0.5.0/src/Makefile.am.wiget	2004-12-29 18:25:27.000000000 +0100
+++ truevision-0.5.0/src/Makefile.am	2004-12-30 16:04:12.327242793 +0100
@@ -4,17 +4,15 @@
 ## If you don't want it to overwrite it,
 ## 	Please disable it in the Anjuta project configuration
 
-INCLUDES = \
-	$(GNOME_CFLAGS)\
-	 -I/usr/include -I/usr/local/include
-
 AM_CFLAGS =\
-	 -Wall\
-	 -O2
+	$(GNOME_CFLAGS)\
+	$(GTKGL_CFLAGS)\
+	 -Wall
 
 AM_CXXFLAGS =\
-	 -Wall\
-	 -O2
+	$(GNOME_CFLAGS)\
+	$(GTKGL_CFLAGS)\
+	 -Wall
 
 bin_PROGRAMS = truevision
 
@@ -73,7 +71,7 @@
 	spline2d.cc
 
 truevision_LDFLAGS = \
-	 -L/usr/lib -L/usr/local/lib -L/usr/X11R6/lib -L../ftgl
+	 -L../ftgl
 
 truevision_LDADD =  \
 	../ftgl/libftgltv.a $(GNOME_LIBS) $(LIBS)
--- truevision-0.5.0/doc/C/Makefile.am.wiget	2004-12-30 16:10:59.115435957 +0100
+++ truevision-0.5.0/doc/C/Makefile.am	2004-12-30 16:10:16.278311927 +0100
@@ -10,5 +10,6 @@
      interface_editor.xml\
      athmospherics.xml
 
+EXTRA_DIST = 
 
 include $(top_srcdir)/doc/truevision-doc.make
--- truevision-0.5.0/configure.in.wiget	2004-12-29 18:25:27.000000000 +0100
+++ truevision-0.5.0/configure.in	2004-12-30 16:04:18.930645307 +0100
@@ -101,19 +101,11 @@
 dnl*************************************************
 dnl GTKGLEXT
 dnl*************************************************
-AC_ARG_WITH(GTKGL-prefix,  [  --with-gtkglext-prefix=DIR    Prefix where GtkGLExt is installed])
-if test "x$with_GTKGL_prefix" = "x" ; then
- GTKGL_LDOPTS="-lgdkglext-x11-1.0"
-  GTKGL_CFLAGS="-I/usr/include/gtkglext-1.0/ -I/usr/local/include/gtkglext-1.0/ -I/usr/lib/gtkglext-1.0/include/ -I/usr/local/lib/gtkglext-1.0/include/"
- else
-  GTKGL_LDOPTS="-L$with_gtkglext_prefix/lib"
-  GTKGL_CFLAGS="-I$with_gtkglext_prefix/include/gtkglext-1.0/"
- fi
- AC_CHECK_LIB(gtkglext-x11-1.0, gtk_gl_init, have_gtkglext=yes, , $LIBS $GNOME_LIBS $GL_LIBS $GTK_LIBS $GTKGL_LDOPTS )
+ PKG_CHECK_MODULES(GTKGL, gtkglext-x11-1.0,,[AC_MSG_ERROR([Missing gtkglext library])])
+ AC_CHECK_LIB(gtkglext-x11-1.0, gtk_gl_init, have_gtkglext=yes, , $LIBS $GNOME_LIBS $GL_LIBS $GTK_LIBS $GTKGL_LIBS )
+
  if test "x$have_gtkglext" = "x" ; then
  	AC_MSG_ERROR([Missing gtkglext library])	
- else
- 	LIBS="$LIBS $GTKGL_LDOPTS -lgtkglext-x11-1.0"
  fi
  
 
@@ -121,15 +113,15 @@
 dnl*************************************************
 dnl Libraries
 dnl*************************************************
-CXXFLAGS="$CXXFLAGS $GTK_CFLAGS $GL_CFLAGS $Z_CFLAGS  $GTKGL_CFLAGS  -I/usr/local/include -I. -I.."
-LIBS="$LIBS $GL_LIBS $Z_LIBS $GTKGL_LDOPTS  -L/usr/local/lib -L/usr/lib -lfreetype"
+CXXFLAGS="$CXXFLAGS $GTK_CFLAGS $GL_CFLAGS $Z_CFLAGS  $GTKGL_CFLAGS -I. -I.."
+LIBS="$LIBS $GL_LIBS $Z_LIBS $GTKGL_LIBS -lfreetype"
 AC_SUBST(CFLAGS)
 AC_SUBST(LIBS)
 
 dnl Languages which your application supports
 ALL_LINGUAS="de fr ru"
 AM_GNU_GETTEXT
-AM_GNU_GETTEXT_VERSION(0.10.40)
+AM_GNU_GETTEXT_VERSION(0.14.1)
 
 dnl Set PACKAGE_LOCALE_DIR in config.h.
 if test "x${prefix}" = "xNONE"; then
@@ -199,7 +191,7 @@
 AC_DEFINE_UNQUOTED(SYSCONFDIR, "${packageprefix}/${packagedatadir}")
 AC_DEFINE_UNQUOTED(LIBDIR, "${packageprefix}/lib")
 
-AC_OUTPUT([
+AC_OUTPUT([ m4/Makefile 
 Makefile
 intl/Makefile
 po/Makefile.in
