--- truevision-0.4.0/src/Makefile.am	2004-04-05 03:43:16.000000000 +0000
+++ truevision-0.4.0.mod/src/Makefile.am	2003-10-15 12:59:21.000000000 +0000
@@ -6,15 +6,7 @@
 
 INCLUDES = \
 	$(GNOME_CFLAGS)\
-	 -I/usr/local/include/gtkglext-1.0 -I/usr/local/lib/gtkglext-1.0/include
-
-CFLAGS =\
-	 -Wall -Wimplicit -Wunused -Wmissing-prototypes\
-	 -O2
-
-CXXFLAGS =\
-	 -Wall -Wimplicit -Wunused -Wmissing-prototypes\
-	 -O2
+	$(GTKGL_CFLAGS)
 
 bin_PROGRAMS = truevision
 
--- truevision-0.4.0/configure.in	2003-10-15 12:59:20.000000000 +0000
+++ truevision-0.4.0.mod/configure.in	2004-04-05 03:33:40.000000000 +0000
@@ -104,33 +104,36 @@
 dnl*************************************************
 dnl GTKGLEXT
 dnl*************************************************
-AC_ARG_WITH(GTKGL-prefix,  [  --with-gtkglext-prefix=DIR    Prefix where GtkGLExt is installed])
-if test "x$with_GTKGL_prefix" = "x" ; then
- GTKGL_LDOPTS="-lgdkglext-x11-1.0"
-  GTKGL_CFLAGS=""
- else
-  GTKGL_LDOPTS="-L$with_gtkglext_prefix/lib"
-  GTKGL_CFLAGS="-I$with_gtkglext_prefix/include"
- fi
- AC_CHECK_LIB(gtkglext-x11-1.0, gtk_gl_init, have_gtkglext=yes, , $LIBS $GNOME_LIBS $GL_LIBS $GTK_LIBS $GTKGL_LDOPTS )
+
+dnl PKG_CHECK_MODULES(GSTUFF, gtk+-2.0 >= 1.3 glib = 1.3.4, action-if, action-not)
+dnl defines GSTUFF_LIBS, GSTUFF_CFLAGS, see pkg-config man page
+dnl also defines GSTUFF_PKG_ERRORS on error
+PKG_CHECK_MODULES(GTKGL, gtkglext-x11-1.0,,exit)
+
+dnl AC_ARG_WITH(GTKGL-prefix,  [  --with-gtkglext-prefix=DIR    Prefix where GtkGLExt is installed])
+dnl if test "x$with_GTKGL_prefix" = "x" ; then
+dnl GTKGL_LDOPTS="-lgdkglext-x11-1.0"
+dnl  GTKGL_CFLAGS=""
+dnl else
+dnl  GTKGL_LDOPTS="-L$with_gtkglext_prefix/lib"
+dnl  GTKGL_CFLAGS="-I$with_gtkglext_prefix/include"
+dnl fi
+dnl i think below is unnessecary
+ AC_CHECK_LIB(gtkglext-x11-1.0, gtk_gl_init, have_gtkglext=yes, , $LIBS $GNOME_LIBS $GL_LIBS $GTK_LIBS $GTKGL_LIBS )
  if test "x$have_gtkglext" = "x" ; then
  	AC_MSG_ERROR([Missing gtkglext library])	
- else
- 	LIBS="$LIBS $GTKGL_LDOPTS -lgtkglext-x11-1.0"
  fi
 
 
 dnl*************************************************
 dnl Libraries
 dnl*************************************************
-CXXFLAGS="$CXXFLAGS $GTK_CFLAGS $GL_CFLAGS $Z_CFLAGS $GTKGL_CFLAGS -I/usr/local/include"
-LIBS="$LIBS $GL_LIBS $Z_LIBS $GTKGL_LDOPTS -L/usr/local/lib"
+CXXFLAGS="$CXXFLAGS $GTK_CFLAGS $GL_CFLAGS $Z_CFLAGS $GTKGL_CFLAGS"
+LIBS="$LIBS $GL_LIBS $Z_LIBS $GTKGL_LIBS"
 AC_SUBST(CFLAGS)
+AC_SUBST(CXXFLAGS)
 AC_SUBST(LIBS)
 
-
-
-
 dnl Set PACKAGE SOURCE DIR in config.h.
 packagesrcdir=`cd $srcdir && pwd`
 
